co-simulation tool/docker/README.md

```diff
@@ -0,0 +1,70 @@
+#  Running CARLA_SUMO_MOSAIC Co-Simulation Tool in Docker 
+
+This is the tutorial to run CARLA_SUMO_MOSAIC co-simulation tool in a docker container. It is recommended to run the co-simulation tool on a **high-performance** computer.
+
+## Test platform
+
+- Operating System: Ubuntu 20.04.2 LTS
+- CPU: Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
+- RAM: 16GB
+- Graphics: NVIDIA GeForce RTX 2070 Super 8GB
+
+### Prerequisite
+
+- NVIDIA GPU with at least 8GB
+- NVIDIA drivers > 361.93
+- Free Disk Space > 30GB
+- NVIDIA Container Toolkit
+- Docker
+
+If you do not install Docker, please see this [link](https://docs.docker.com/engine/install/ubuntu/).
+
+To install NVIDIA Container Toolkit, please see this [link](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html).
+
+### Build Docker Image of Co-simulation Tool
+
+#### Preparation for building docker image
+
+- Put the docker file (Dockerfile), the folder of CARLA executable file (CARLA_0.9.10), the folder of Co-simulation tool bundle (Carla_Sumo_Mosaic) in the same folder.
+- Change the CARLA path in the CARLA configuration file of scenarios to the location of CARLA executable file
+- The .sh files (CarlaUE4.sh and bridge.sh) must be given permission
+
+#### Build docker image
+
+##### step 1:  Run the following command to build the docker image
+
+```
+$ docker build - < Dockerfile -t carla-sumo-mosaic
+```
+
+##### step 2: copy CARLA and Co-Simulation tool to the docker image and commit them
+
+Copy `CARLA_0.9.10` and `Carla_Sumo_Mosaic` folder into this container.
```

Comment:
Remind users to change the carlaUE4Path in carla_config.json file under the scenarios folder before copy Carla_Sumo_Mosaic folder into container.

Reply:
The commands to setup environment variable CARLA_HOME have been added to dockerfile.